core/reader_re2.f

```diff
@@ -90,10 +90,10 @@ subroutine readp_re2_mesh(ifbswap,ifread) ! version 2 of .re2 reader
       common /ctmp1/  vi
 
       integer*8       lre2off_b,dtmp8
-      integer*8       nrg
+      integer*8       nrg      
 
       nrg       = nelgt
-      nr        = nelt
+      nr        = nelt  ! 09/03/2022, supports only up to 2B elements
```

Comment:
I find this comment confusing. The 2B element restriction isn't introduced here.

Reply:
nr is not integer*8 in this routine and so it was a reminder note for future changes, but not necessary to keep it here. We can remove.

Reply:
`nr` stores the local number of "records" (here elements) to read. No need for `integer*8`

Reply:
Yes.

Reply:
nr        = nelt  ! 09/03/2022, supports only up to 2B elements
      irankoff  = igl_running_sum(nr) - nr

The note was for the next line -- eventually needing irankoff8 = i8gl_running_sum(nr) - nr
with nr=nelt where nr integer*8.

Reply:
Got it. Indeed this has to change but there are many other locations which need to be changed too. So having a comment just for this line can be misleading. Anyway it's a minor issue.

Reply:
Yes too many places to change accordingly once int*8 applied, and so we kept the routine as they were, but leaving a small hint to remember. We can change the commit as preferred or I'm fine to remove.

Reply:
I suggest to remove the comment but create an detailed GitHub issue what remaining issue we need to fix.